console/src/components/Main/Main.js

```diff
@@ -776,97 +765,23 @@ class Main extends React.Component {
                 {renderProPopup()}
               </div>
               <Link to="/settings">
-                <div className={styles.helpSection + ' ' + styles.settingsIcon}>
-                  {getMetadataStatusIcon()}
-                  {getSettingsSelectedMarker()}
+                <div className={styles.supportSection}>
+                  <div className={styles.helpSection}>
```

Comment:
What about _merging_ it into one div:
```
<div className={`${styles.supportSection} ${styles.helpSection}`}>
```
?

Reply:
This was due to the way the current css was structured and I didnt want to go down that hole. Updated it though.